momo.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8">
+
+    <title>Momo Time!</title>
+
+    <link rel="stylesheet" type="text/css" href="styles.css">
```

Comment:
```suggestion
    <link rel="stylesheet" type="text/css" href="css/momostyle.css">```

make ur own style first